Plugins/Flow.Launcher.Plugin.Explorer/Search/ResultManager.cs

```diff
@@ -39,25 +41,36 @@ internal static Result CreateFolderResult(string title, string subtitle, string
                             return false;
                         }
                     }
+                    // one of it is enabled
+                    var keyword = Settings.EnabledSearchActionKeyword ? Settings.SearchActionKeyword : Settings.PathSearchActionKeyword;
+
+                    keyword = keyword == Query.GlobalPluginWildcardSign ? string.Empty : keyword + "";
 
                     string changeTo = path.EndsWith(Constants.DirectorySeperator) ? path : path + Constants.DirectorySeperator;
-                    Context.API.ChangeQuery(string.IsNullOrEmpty(query.ActionKeyword) ?
-                        changeTo :
-                        query.ActionKeyword + " " + changeTo);
+                    Context.API.ChangeQuery($"{keyword}{changeTo}");
```

Comment:
You removed the space here, is it meant to be added to line 47?

Reply:
Yes, it is supposed to be a space instead of empty string. Will fix it soom.